chain/validation/lotus_test.go

```diff
@@ -0,0 +1,136 @@
+package validation
+
+import (
+	"encoding/binary"
+	"math/big"
+	"testing"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	mh "github.com/multiformats/go-multihash"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/chain-validation/pkg/state"
+	"github.com/filecoin-project/chain-validation/pkg/suites"
+
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/address"
+)
+
+func TestStorageMinerValidation(t *testing.T) {
+	factory := NewFactories()
+	suites.CreateStorageMinerAndUpdatePeerIDTest(t, factory)
+
+}
+
+func TestValueTransfer(t *testing.T) {
+	factory := NewFactories()
```

Comment:
can we move calling these to chain/vm/validation_test.go ?

Reply:
Yes I believe so, need to rework a few dependencies since the [validation driver depends on the lotus vm](https://github.com/filecoin-project/lotus/pull/539/files#diff-173eebafb6fcc6d6702e21f1d357a334R12).